src/TLuaInterpreter.cpp

```diff
@@ -2563,6 +2563,14 @@ int TLuaInterpreter::scrollingActive(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#holdingCtrl
+int TLuaInterpreter::holdingCtrl(lua_State* L)
```

Comment:
What do you think about making it generic for all modifiers? Something that makes use of `mudlet.keymodifier.` and also allows you to test for a combination of modifiers. 

See https://wiki.mudlet.org/w/Manual:Lua_Functions#permKey for an example.

Reply:
Yes, that feels like the correct way to do it. I'll update it.